src/robot/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#==============================================================================
```

Comment:
In order to put this file into the project, we need to add `add_subdirectory("robot")` to `src/CMakeLists.txt`.